### DISCUSSION TITLE
InitialContextSetupRequest

### DISCUSSION THREAD

**Discussion by ganeshg87** [aligungr/UERANSIM#358] [Q&A]
Can UeRanSim Trigger UEContextReleaseRequest ?

**Comment by aligungr** [comment]
@ganeshg87 

Yes sure, you can use `nr-cli` to trigger a UEContextReleaseRequest.

**Reply by ganeshg87** [reply]
vagrant@ue-ran-sim:~/UERANSIM/config/scripts$ ../../build/nr-cli imsi-901700000000001
$ commands
info           | Show some information about the UE
status         | Show some status information about the UE
timers         | Dump current status of the timers in the UE
rls-state      | Show status information about RLS
coverage       | Dump available cells and PLMNs in the coverage
ps-establish   | Trigger a PDU session establishment procedure
ps-list        | List all PDU sessions
ps-release     | Trigger a PDU session release procedure
ps-release-all | Trigger PDU session release procedures for all active sessions
deregister     | Perform a de-registration by the UE


could you please direct me, how can I trigger UEContextReleaseRequest from nr-cli?**

**Reply by aligungr** [reply]
Explicit release of NAS signalling is triggered by gNodeB, but not UE.

So please check the gNodeB commands instead of UE commands.

**Reply by ganeshg87** [reply]
yes, got it.

thanks a lot @aligungr
